lib/AST/ASTVerifier.cpp

```diff
@@ -3202,7 +3202,8 @@ bool swift::shouldVerify(const Decl *D, const ASTContext &Context) {
     return true;
   }
 
-  size_t Hash = llvm::hash_value(VD->getNameStr());
+  size_t Hash =
+      llvm::DenseMapInfo<DeclBaseName>::getHashValue(VD->getBaseName());
```

Comment:
Please use a hash that's stable from run to run. (The AST verifier doesn't need to be fast.)